js/views/pauseScreen.js

```diff
@@ -109,52 +149,70 @@ var PauseScreen = createReactClass({
         }
       }, 1000);
     }
-  },
+  };
 
-  handlePlayerMouseDown: function(e) {
+  handlePlayerMouseDown = (e) => {
     if (this.props.controller.videoVr) {
       e.persist();
     }
     this.props.controller.state.accessibilityControlsEnabled = true;
     this.props.controller.state.isClickedOutside = false;
     this.props.handleVrPlayerMouseDown(e);
-  },
+  };
 
-  handlePlayerMouseMove: function(e) {
+  handlePlayerMouseMove = (e) => {
     this.props.handleVrPlayerMouseMove(e);
-  },
+  };
 
   /**
    * call handleVrMouseUp when mouseup was called on selectedScreen
    * @param {Event} e - event object
    */
-  handlePlayerMouseUp: function(e) {
+  handlePlayerMouseUp = (e) => {
     e.stopPropagation(); // W3C
     e.cancelBubble = true; // IE
-  },
+  };
 
   /**
    * call handleVrMouseUp when mouseup was called on document
    * @param {Event} e - event object
    */
-  handleVrMouseUp: function(e) {
+  handleVrMouseUp = (e) => {
     this.props.handleVrPlayerMouseUp(e);
-  },
+  };
 
   /**
    * Make sure keyboard controls are active when a focusable element has focus.
    *
    * @param {Event} event - Focus event object
    */
-  handleFocus: function(event) {
+  handleFocus = (event) => {
     var isFocusableElement = event.target || event.target.hasAttribute(CONSTANTS.KEYBD_FOCUS_ID_ATTR);
     if (isFocusableElement) {
       this.props.controller.state.accessibilityControlsEnabled = true;
       this.props.controller.state.isClickedOutside = false;
     }
-  },
+  };
 
-  render: function() {
+  /**
+   * MouseLeave event handler. Hides the pause screen if the autoHide skin config is set to true.
+   * @private
+   */
+  handleMouseLeave = () => {
```

Comment:
Would there be a way for the controls to auto hide when the video is in fullscreen mode since the mouse can't leave the container?

Reply:
This sounds like a product question. I'll bring it up with Steve.

Reply:
Ok thanks. If he does want to enable that we could probably re-use the logic from the playing screen, it already handles this scenario

Reply:
It looks like Amazon changed their behavior. Steve would like us to mimic the new one, which is autohide after 1.5 seconds if the mouse is not over one of our buttons.